src/olympia/abuse/tests/test_cinder.py

```diff
@@ -1510,19 +1512,19 @@ def test_workflow_recreate_no_versions_to_flag(self):
             json={'job_id': '2'},
             status=201,
         )
-        assert cinder_instance.workflow_recreate(job=cinder_job) == '2'
+        assert cinder_instance.workflow_recreate(notes='', job=cinder_job) == '2'
```

Comment:
In code land, it seems like the only real use case for worfklow_recreate is to pass the notes from the cinder_job as the notes argument. Is it possible that this property is None or otherwise false? What happens in this case? Might be worth a test scenario if you opt not to read that arg from the job argument (see comment above)

Reply:
So then passing None should just pass right? Probably not a bad idea to add that then either here or in a separate test. That way the test is trying it's hardest to fail and doesn't because our code is so awesome.